### PR TITLE
Change references to GYR Express to CTC

### PR DESCRIPTION
**Pull request by embarnard**

### CODE REVIEW

**Comment by bytheway875** [comment]
@alaroia-cfa Mentioned something yesterday about needing to update changes to Spanish translations on Transifex directly. @alaroia-cfa @jenny-heath can one of ya'll weigh in on what needs to happen there for something like this?

**Comment by embarnard** [comment]
Hmmm not sure what happened here that it would include all these other commits.. maybe i need to git rebase -i and just pick the relevant commits ... looking into it

File: app/views/questions/triage/_ctc_tile.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="marketing-comparison">
   <h2><%= t('.title') %></h2>
   <p><%= t('.description') %></p>
-  <div class="timeframe express">
```
Comment (embarnard) [inline app/views/questions/triage/_ctc_tile.erb:4]: 👩‍🍳

**Review by embarnard — COMMENTED** [review]
approved by al

**Comment by alaroia-cfa** [comment]
@embarnard Good catch on that final commit! (I approve that too.)

**Pull request merged**